plugins/modules/helm.py

```diff
@@ -502,7 +514,12 @@ def main():
             # when deployed without an 'appVersion' chart value the 'helm list' command will return the entry `app_version: ""`
             appversion_is_same = (chart_app_version == released_app_version) or (chart_app_version is None and released_app_version == "")
 
-            if force or release_values != release_status['values'] \
+            if values_files:
+                values_are_good = release_status['values'] == load_values_files(values_files)
```

Comment:
Only nit with this patch is this variable name—it's not like if they don't match they're _bad_. I would rather this variable be named `values_match` or something along those lines.

Reply:
indeed. Good point.